openshift/certificate_renewal.adoc

```diff
@@ -0,0 +1,142 @@
+= OCP Certificate Renewal
+Snowdrop Team (Antonio Costa)
+:icons: font
+:revdate: {docdate}
+:toc: left
+:toclevels: 3
+:description: Renew OCP cluster certificate
+ifdef::env-github[]
+:tip-caption: :bulb:
+:note-caption: :information_source:
+:important-caption: :heavy_exclamation_mark:
+:caution-caption: :fire:
+:warning-caption: :warning:
+endif::[]
+
+This document describes the process to renew an OCP cluster certificates.
+
+== References
+
+* https://docs.openshift.com/container-platform/4.10/security/certificates/replacing-default-ingress-certificate.html
+
+== Obtain the Issuer certificate
```

Comment:
The text to be documented here should be something like:
"By default, the certificate issued during the installation of the OpenShift cluster on RHOS do not include the DNS names defined to access the console, API, oauth, etc services. In this case, it is needed to use a Certificate authority like Letsencrypt to trust a certificate created for different DNS names and using a domain name hosted by a DNS provider (godaddy, cloudfare, etc).

such a process can be managed by the cert-manager when we issue a certificate CR packaging the ACME DNS01 solver and able to contact the DNS provider to validate if the DNS names to be used are valid/exist.

IF this is the case, then letsencrypt will issue a Certificate for the DNS names and will include part of the certificate their CA root certificate R3 

By doing that, the worker nodes will be able to access the API server running on the masters, the users will not get anymore a certificate not valid from their browser if of course you following the instruction detailed here: https://docs.openshift.com/container-platform/4.12/security/certificates/replacing-default-ingress-certificate.html

Reply:
updated the docs.